src/plugins/controls/public/react_controls/control_group/components/control_panel.scss

```diff
@@ -27,6 +27,7 @@
   .controlErrorButton {
     width: 100%;
     border-radius: 0 $euiBorderRadius $euiBorderRadius 0 !important;
+    height: $euiButtonHeightSmall;
```

Comment:
This was done to be consistent with below `optionsList` styles.

https://github.com/elastic/kibana/blob/fabe07d94ed5a7fa90a7e5e14f8806607ddf543a/src/plugins/controls/public/options_list/components/options_list.scss#L9-L12

Reply:
Awesome thank you!